### ISSUE TITLE
Some Issues

### ISSUE THREAD

**Issue by Deathevan** [V0idhead/WOTRNineSwords#5]
Hey, it's me again. Playing my campaign (so far gamebreaking bugless) since beginning with your class, so expect a lot of feedback. Here it goes:

Battle Clarity, Battle Ardor, and Battle Skill aren't working, Battle Cunning gives Attack instead of a Damage modifier, Bounding Assault teleports properly but doesn't give the +2 on attack, and there's a possible issue with Hearing the Air.

The first screenshot shows Reflex Saves and the second shows both the critical confirmation roll and Battle Cunning adding the +5 Insight Bonus to Attack.

 The third screenshot shows the damage for the same attack shown on the second screenshot (just to be thorough); I used Ruby Nightmare Blade and its extra damage hasn't added it either, although I have no clue whether it should (fourth screenshot).

As for Battle Skill, the CMD shown on the fifth screenshot is 10 BAB+ level 10 + 1 Str + 6 Dex + 1 from Engraved Lucky Bracers (yea this one took me a while to find out).

Punishing Stance gives +4 damage instead of 1d6 (Sixth screenshot).

Bounding Assault teleports properly with rare exceptions (due to engaged enemies or line of sight), but I don't see it adding the +2 attack (it gives no specific combat log messages so no screenshot).

Lastly, the slightly tougher one: Hearing the Air gives the intended +5 Perception, but Blindsight doesn't work on Mirror Image, although It seems to be working against concealment (I've only managed to land two attacks on one enemy so far, freaking Minotaur Flame Struck its own buddy lol). I tried looking it up but each Tabletop or Wrath of the Righteous website and discussion board says a different thing about Blindsight, so my bet is that it's WAI in game.

I've only seriously tried one build so far so I haven't tested even half the maneuvers, but up to this moment I don't recall anything wrong. Since I'm trying a Dex build I've been avoiding the pure Str checks, but as far as I know the lower level maneuvers with Strength checks, as well as those with Athletics checks, are working. Of course, as of this post, I haven't gone past level 10 so there's still some ground to cover.

Thanks again for the amazing job!

![image](https://github.com/user-attachments/assets/310fb913-6085-47c9-897b-f8fdf9595ce2)

![image](https://github.com/user-attachments/assets/72899664-9a80-4c59-a3ba-d720eadafeee)

![image](https://github.com/user-attachments/assets/801823cb-d9ab-40f0-af21-7142fb8c8012)

![image](https://github.com/user-attachments/assets/160e0ae8-1fb3-4257-b793-eaa90de50c83)

![image](https://github.com/user-attachments/assets/97ec60b0-9361-4b45-897f-0a19c5eb5776)

![image](https://github.com/user-attachments/assets/85c56101-f9c9-4d6e-a38e-e86e7b48b464)

Also, I'm sorry if I shouldn't have posted everything at once, but I'm not really used to GitHub.

**Comment by V0idhead** [comment]
Thanks for your attention to detail that I apparently lack.

I found and fixed the bugs in BattleArdor and BattleClarity.

Battle Skill is awared at level 11, it seems to work correctly on my 2 test-characters.

I'll think about Bounding Assault, I forgot about the +2 to attack during testing because it's substantially changed (and way more powerful already).

Punishing Stance is similar, I couldn't get the 1d6 to work, so I averaged it to +4 then forgot about it while I worked on something else, I'll look into it, but it may be a while until I get around to it.

I'll work on Battle Cunning (which is a bit more complicated) then release a new version.

Putting the different problems into different GitHub-issues makes it easier to to keep track of the current status of  each problem.
But don't worry about it, I'm absolutely thrilled to have someone other than me test everything.
